devel/README.md

```diff
@@ -271,6 +279,39 @@ tooling functions correctly
 (You can safely ignore some errors like
 `com_github_opencontainers_go_digest Does not exist`)
 
+### Building the containers
+
+This tool builds and publishes Mixer container images to the specified
+registry.
+
+```
+bin/publish-docker-images.sh -h gcr.io/my-project -t my-tag
+```
+
+where
+
+* The `-h` parameter `gcr.io/my-project` is the composition of the registry
+  hostname and the project id. This should be customized.
+* The `-t` parameter `my-tag` is the desired tag. This should be customized.
+
+### Building the Istio manfiests
+
+Use [updateVersion.sh](https://github.com/istio/istio/blob/master/install/updateVersion.sh)
+to generate new manifests with the specified Mixer containers.
+
+```
+cd $ISTIO/istio
+install/updateVersion.sh -rgcr.io/my-project,tag
```

Comment:
mixer is -x afaik - did you test (run the command, followed by git diff istio.VERSION ?)
also it's best if you add commits vs force pushing updates so the preview review don't disappear

Reply:
@ldemailly I did test with git diff (this is my standard workflow) - however it was a groggy ~2am (my teenagers woke me up...) so missed the fact that this was modifying proxy not mixer.

Re force pushing will do in the future - the current workflow states to force push - however the workflow doesn't address the case of making a change to a PR.  Will do additional commits in the future.

Fix incoming.

Reply:
oh can you edit wherever it says to force push, that's bad after the PR was once submitted

that's why we use "squash and merge" so in master there will be the final product but the PR should have the iterations